package.json

```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "start": "if-env NODE_ENV=production && npm run start:prod || npm run start:dev",
-    "start:dev": "webpack-dev-server --inline --content-base public/ --history-api-fallback",
-    "start:prod": "webpack && node server.js"
+    "start:dev": "webpack-dev-server --inline --hot --content-base public/ --history-api-fallback",
```

Comment:
gotcha, hmm I thought that this was already happening though. Like if I am running the dev server, if I make a change in the text editor and save, it will automatically refresh the page. Is it not doing that for you?